index.html

```diff
@@ -2878,17 +2879,24 @@ <h2>
       </h2>
 
       <p>
-If a <a>DID method</a> specification is written for a public <a>DID registry</a>
-where all <a>DIDs</a> and <a>DID documents</a> are publicly available, it is
-<em>critical</em> that <a>DID documents</a> contain no PII. All PII should be
-kept behind <a>service endpoints</a> under the control of the
-<a>DID subject</a>. With this privacy architecture, PII can be exchanged on a
-private, peer-to-peer basis using communications channels identified and secured
-by <a>public key descriptions</a> in <a>DID documents</a>. This also enables
-<a>DID subjects</a> and relying parties to implement the
-<a href="https://en.wikipedia.org/wiki/General_Data_Protection_Regulation">GDPR</a>
-<a href="https://en.wikipedia.org/wiki/Right_to_be_forgotten">right to be forgotten</a>,
-because no PII is written to an immutable <a>distributed ledger</a>.
+        If a <a>DID method</a> specification is written for a public <a>DID registry</a>
+        where all <a>DIDs</a> and <a>DID documents</a> are publicly available, it is
+        <em>critical</em> that <a>DID documents</a> contain no PII. All PII should be
+        kept behind <a>service endpoints</a> under the control of the <a>DID
+          subject</a>. Additional due diligence should be taken around the use of URLs
+        in service endpoints as well to prevent leakage of unintentional PII or
+        correlation within a URL of a service endpoint. For example, a URL that contains
+        a username is likely dangerous to include in a <a>DID Document</a> because the
+        username is likely to be human human-meaningful in a way that can
+        unintentionally reveal information that the <a>DID Subject</a> did not consent
+        to sharing. With this privacy architecture, PII can be exchanged on a private,
+        peer-to-peer basis using communications channels identified and secured by
+        <a>public key descriptions</a> in <a>DID documents</a>. This also enables <a>DID
+          subjects</a> and relying parties to implement the<a
+          href="https://en.wikipedia.org/wiki/General_Data_Protection_Regulation">GDPR</a><a
+          href="https://en.wikipedia.org/wiki/Right_to_be_forgotten">right to be
+          forgotten</a>, because no PII is written to an immutable <a>distributed
+          ledger</a>.
```

Comment:
```suggestion
        <a>public key descriptions</a> in <a>DID documents</a>. This also enables <a>DID
        subjects</a> and relying parties to implement the <a
        href="https://en.wikipedia.org/wiki/General_Data_Protection_Regulation">GDPR</a> <a
        href="https://en.wikipedia.org/wiki/Right_to_be_forgotten">right to be
        forgotten</a>, because no personal data is written to an immutable <a>distributed
        ledger</a>.
```